packages/kit/test/apps/aa-routing/src/app.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
rather than creating a new top-level app, could we move these into `test/apps/options`? it will mean updating the paths in the existing tests in that app, but i think that's preferable to creating a new app, since that will have a noticeable impact on how long tests take to run overall